### ISSUE TITLE
Improved Baselines with Momentum Contrastive Learning 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#11]
# TL;DR
Moco(https://arxiv.org/abs/1911.05722) has been improved by using Using "MLP projection head", "strong data augmentation” used in SimCLR which is leading represetntation learning method.

# Why it matters:

# Paper URL
https://arxiv.org/abs/2003.04297

# Submission Dates(yyyy/mm/dd)
2020/03/09

# Authors and institutions
Xinlei Chen, Haoqi Fan, Ross Girshick, Kaiming He
- Facebook AI Research

# Methods

# Results
Both MLP heads and  strong augmentations improve MoCo(v1).
![image](https://user-images.githubusercontent.com/24524018/89390355-8c01dd80-d741-11ea-983e-a5e4e1de5a86.png)

# Comments
